spec/models/piece_spec.rb

```diff
@@ -94,4 +94,37 @@ def create_chess(x, y)
       end
     end
   end
+
+  describe '#on_board?' do
+    it 'should return true is the coords are from 1 to 8' do
+      piece = Piece.create
+      (1..8).each do |x|
```

Comment:
1 new line will be fine.

2 it will complain.

I have a good article on this somewhere (with code readability tips from someone much smarter than me, I'll dig it up) 😄